src/index.ts

```diff
@@ -126,7 +126,11 @@ function activateLatexPlugin(app: JupyterLab, manager: IDocumentManager, editorT
       pdfContext.disposed.connect(cleanupPreviews);
     };
 
-    const errorPanelInit = () => {
+    const errorPanelInit = (err: ServerConnection.ResponseError ) => {
+      if (err.response.status === 404) {
+        throw err;
```

Comment:
It may be nice to also add a more descriptive `console.error` suggesting people check the serverextension install here.